modules/virt-example-node-placement-tolerations-hyperconverged-cr.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/install/virt-specifying-nodes-for-virtualization-components.adoc
+
+[id="virt-example-node-placement-tolerations-hyperconverged-cr_{context}"]
+= Example: Node placement with tolerations in the HyperConverged Cluster CR
+
+In this example, nodes that are intended for virtualization resources are labeled with the `key=virtualization:NoSchedule` taint. Only pods with the matching tolerations are scheduled to these nodes.
```

Comment:
s/intended/reserved/

I think that the language used in https://github.com/openshift/openshift-docs/pull/26966/files#diff-af352d2ba00b7510b0f61efe50896fb729e96d29dccfd72a59120805c7f59565R8  ("reserved") better describes what taints do.

Reply:
Updated. Thanks for reviewing, @dankenigsberg!